praw/models/reddit/subreddit.py

```diff
@@ -17,6 +17,117 @@ class Subreddit(RedditBase, MessageableMixin, SubredditListingMixin):
     STR_FIELD = 'display_name'
     MESSAGE_PREFIX = '#'
 
+    @staticmethod
+    def _create_or_update(_reddit, allow_images=None, allow_top=None,
+                          collapse_deleted_comments=None,
+                          comment_score_hide_mins=None, description=None,
+                          domain=None, exclude_modqueue_banned=None,
+                          header_hover_text=None, hide_ads=None, lang=None,
+                          key_color=None, link_type=None, name=None,
+                          over_18=None, public_description=None,
+                          public_traffic=None, show_media=None,
+                          spam_comments=None, spam_links=None,
+                          spam_selfposts=None, sr=None, submit_link_label=None,
+                          submit_text=None, submit_text_label=None,
+                          subreddit_type=None, suggested_comment_sort=None,
+                          title=None, wiki_edit_age=None, wiki_edit_karma=None,
+                          wikimode=None):
+        """Create a new subreddit.
+
```

Comment:
With moving the parameter explanation to other functions, I don't think this docstring is necessary on this function.